templates/dashboard.html

```diff
@@ -4,9 +4,9 @@ <h2>Dashboard</h2>
 
 {{ template "snippets/profile.html" .Profile }}
 
-<p>
+<p class="submenu">
     <a class="btn" href="profile/{{ .Profile.GetAddressString }}">View Your Profile</a>
-    <a class="btn" href="memo/new">New Memo</a>
+    <a class="btn new_memo" href="memo/new">New Memo</a>
```

Comment:
Are you using this for styling or for JS selection?

Reply:
It was intended so we could move the new memo button up into the corner of the screen like twitter does, and then if I felt extra energetic I'd probably make a popup for quick-memo.

At this point though I think it would be better to just do on the page itself (see https://share.dmca.gripe/wK3pTvsBsAWMgXlE.png), which I'll probably push in a separate PR if/when I have some free time to 1) make it look more polished, 2) implement it in actual css/html